includes/DbService.php

```diff
@@ -4,17 +4,24 @@
 
 use SD\Sql\PropertyTypeDbInfo;
 use SD\Sql\SqlProvider;
+use SMW\MediaWiki\Connection\Database;
 use Wikimedia\Rdbms\DBConnRef;
 use Wikimedia\Rdbms\IResultWrapper;
 
 class DbService {
 
 	private DBConnRef $dbw;
 	private DBConnRef $dbr;
+	private DBConnRef $smwDbr;
+	private Database $dbrQueryEngine;
 
 	public function __construct( ?DBConnRef $dbw, ?DBConnRef $dbr ) {
 		$this->dbw = $dbw;
 		$this->dbr = $dbr;
+		// Fandom change - obtain a connection to the external SMW cluster
+		$this->smwDbr = smwfGetStore()->getConnection( DB_REPLICA );
```

Comment:
Sometimes we use `getConnection( 'mw.db' )` and sometimes this form. What is the difference?